app/react/Documents/components/ShowToc.js

```diff
@@ -7,6 +7,8 @@ import ShowIf from 'app/App/ShowIf';
 import { t } from 'app/I18N';
 import { Icon } from 'UI';
 
+import './scss/showToc.scss';
```

Comment:
Have we moved away from creating scss files in the App folder namespaced by either component type or section type?  Also, usually scss files are imported in a single place instead on a per-component basis.  Is this actually working?  We don't really have styled components, which would be ideal.  But not sure this is the right approach?

Reply:
im not sure either, this definitely works, and it also make sense, the css go with the component, if you remove this component this css will not be part of the final build instead of having to remember that this have some associated specific css.